contents/handbook/growth/billing.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Billing
+sidebar: Handbook
+showTitle: true
+---
+
+## Setting up billing
+For customers with special pricing (i.e. very large volumes or Enterprise & Supported plans), we need to manually set up the billing information on the system. This page contains instructions for setting up billing. Please note this page covers the process after an official PostHog quote has been approved by the customer. For information before this stage, please refer to the [Sales](/handbook/growth/sales) section of the handbook.
+
+### Self-hosted
+Contrary to cloud plans, **all self-hosted _paid_ plans must be manually prepared** (i.e. there's no self-serve option). To set up billing for self-hosted, please follow these instructions:
+
+#### Pre-setup
+This process only needs to happen once.
+1. Download the Postman collection from [license][license].
```

Comment:
What does 'Download the Postman collection' mean?

Reply:
Resolving this as we'll have a call tomorrow for discuss.